bitcoin/core/script.py

```diff
@@ -1061,4 +1065,4 @@ def SignatureHash(script, txTo, inIdx, hashtype):
         'RawSignatureHash',
         'SignatureHash',
         'IsLowDERSignature',
-)
+)
```

Comment:
@epheph absolutely, I didn't mean to do that. I'm also not very experienced with git and interested in learning, what would be the best way to revert?